nbgitpuller/static/index.js

```diff
@@ -42,10 +36,10 @@ require([
 
 
     GitSync.prototype.start = function() {
+        // Start git pulling handled by SyncHandler, declared in handlers.py
         var syncUrl = this.baseUrl + 'git-pull/api?' + $.param({
             repo: this.repo,
-            branch: this.branch,
-            path: this.path
```

Comment:
Just to clarify, this.path was unused in the handler, right? And the `subPath` param is handled elsewhere?

Reply:
Yepp - SyncHandler defined in handlers.py had no use of the path being passed to it, it is focused on a git sync that only cares about the repo and branch.

I moved some logic from index.js to the UIHandler in handlers.py.